openSUSE-2017.7.2/salt.spec

```diff
@@ -571,12 +715,21 @@ install -Dpm 0644 pkg/fish-completions/* %{buildroot}%{fish_completions_dir}
 
 %if 0%{?suse_version} > 1020
 %fdupes %{buildroot}%{_docdir}
+%if 0%{?build_py2}
+%fdupes %{buildroot}%{python_sitelib}
+%endif
+%if 0%{?build_py3}
 %fdupes %{buildroot}%{python3_sitelib}
 %endif
+%endif
 
 %check
 %if %{with test}
+%if 0%{?default_py3}
 python3 setup.py test --runtests-opts=-u
+%else
+python setup.py test --runtests-opts=-u
```

Comment:
I just noticed here that we're calling python directly. Should we use `%{__python}` here? Same in L729 but `%{__python3}`.

Reply:
done